website/docs/tutorials/08-formatting.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_label: Formatting
+---
+
+# Formatting
+
+Protostar comes with a built-in Cairo source code formatter.
+
+```shell
+protostar format
```

Comment:
If i run it like this what files it will format? Would be nice to include such infromation

Reply:
That information is included withing CLI Reference, I don't think we need to duplicate it, perhaps
```suggestion
protostar format --help
```
is more adequate?